donkstation/code/datums/diseases/advance/symptoms/rodman.dm

```diff
@@ -0,0 +1,115 @@
+/datum/symptom/rodman_syndrome
```

Comment:
Wasn't able to reproduce any effects of rodman, not sure why

Reply:
If you are var-editing the stats on the virus itself, that does not actually change the variables inside the symptom.
You'll want to modify "restore_mind", "increase_volume" and "awaken_monkey" manually